assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/gencerts.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
Why is this change needed?

Reply:
I think I added it because we moved away from alpine to fedora, and bash being readily available in fedora, I thought might as well switch to it. But it turns out `/bin/sh` is a sym link to `/bin/bash`. So I guess it is fine.